lua/entities/kleiner/sv_spawning.lua

```diff
@@ -0,0 +1,93 @@
+﻿-- This file is subject to copyright - contact swampservers@gmail.com for more information.
+-- INSTALL: CINEMA
+CreateConVar("kleiner_spawncount", "18", FCVAR_ARCHIVE + FCVAR_PROTECTED, "Number of Kleiner NPCs to spawn automatically.", 0, 100)
+KLEINER_NPCS = KLEINER_NPCS or {}
+KLEINER_NPCS_FILTER = KLEINER_NPCS_FILTER or {}
+KLEINER_NPC_TARGETS = KLEINER_NPC_TARGETS or {}
+KLEINER_NPCS_CURRENT_NUMBER = KLEINER_NPCS_CURRENT_NUMBER or 0
+KLEINER_DESIRED_NUMBER = KLEINER_DESIRED_NUMBER or GetConVar("kleiner_spawncount"):GetInt() or 0
+
+function KLEINER_NPC_SETUPTIMER(value)
```

Comment:
this is overcomplicated and doesn't work when i change the convar. Just do `timer.Create` in the global scope (eg. not inside a function) and check the `GetConVar("kleiner_spawncount"):GetInt()` every time. There is no reason to use `hook.Add("Initialize"`; just put it in the global scope.